test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -110,14 +111,21 @@ def inline_flatten_list(inputs, res_list):
 
 def run_ort(ort_sess, input):
     input = flatten_tuples(input)
-    input = to_numpy(input)
+    input_final = []
```

Comment:
Can we take this opportunity to share the flatten + unpack + `to_numpy` between `run_ort` and `ort_compare_with_pytorch`?

The only difference seems to be `flatten_tuples` vs `torch.jit._flatten`. My guess is it's safe to use `torch.jit._flatten` in both cases.